test/integration/suite_test.go

```diff
@@ -9,36 +9,185 @@ import (
 	"os"
 	"testing"
 
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
```

Comment:
I'm pretty unfamiliar with this tool but it sure does make testing easier :)

Reply:
Yeah that's why it exists. I think there's a chance we'd be open to giving it to upstream, but I'm pretty sure the maintenance of trying to be "the testing framework for kubernetes" would be intense.